src/repository/storage/postgres/orm/sequelize/noteRelations.ts

```diff
@@ -209,4 +210,37 @@ export default class NoteRelationsSequelizeStorage {
 
     return foundNote !== null;
   };
+
+  /**
+   * Get all parent notes of a note that a user has access to,
+   * where the user has access to.
+   * @param noteId - the ID of the note.
+   */
+  public async getNoteParentsIds(noteId: NoteInternalId): Promise<NoteInternalId[]> {
+    // get all note ids via a singe sql query instead of many
+    const query = `
+    WITH RECURSIVE note_parents AS (
+      SELECT np.note_id, np.parent_id
+      FROM ${String(this.database.literal(this.tableName).val)} np
+      WHERE np.note_id = :startNoteId
+      UNION ALL
+      SELECT nr.note_id, nr.parent_id
+      FROM ${String(this.database.literal(this.tableName).val)} nr
+      INNER JOIN note_parents np ON np.parent_id = nr.note_id
+    )
+    SELECT np.note_id, np.parent_id
+    FROM note_parents np;`;
```

Comment:
for propper naming we will need 
```sql
    SELECT np.note_id as noteId, np.parent_id as parentId
    FROM note_parents np;`;
```
however in sequelize config we've set `underlinde: true`, but in case you are working with database directly, without `model` connection you can also try to add `underline: true` to the actual query request below (setting `underline` parameter would be optimal)

This is needed to get proper entity with camelCase naminig as a response instead of just disabling eslint naming convention